### ISSUE TITLE
Reference Loop and Formulas wont calculate

### ISSUE THREAD

**Issue by MaxMorais** [jspreadsheet/ce#1030]
Hi, first of all, thank you so much for that amazing product!

I'm using jexcel for our timesheet control, at company
Basically it's composed by 2 instances of Jexcel, where the first instance update the second instance, it works fine

But the second instance, is composed, by some areas editable, and some areas that are only calculated formulas.
I'm receiving on the browser console a message of reference loop, but really I'm not able to understand what's happening

![TS 2020 00559](https://user-images.githubusercontent.com/1288676/85560287-e49d7100-b600-11ea-9d6e-03cd05ea78e8.png)

This is a visualization of the timesheet.
Just the last line dont get computed

Here I do have a implementation of all formulas I'm using

https://docs.google.com/spreadsheets/d/1owNikOq_5heR58TTjMlUHIhATJh8RHtxMxIQJs3KTjI/edit?usp=sharing

This is the same group of formulas

Basically, the reference error, start to be reported on line 14

If you accept any kind of sponsor, for get this issue fixed, pls, let me know at mmorais@aptitudetech.net

**Comment by pphod** [comment]
I will take a look on your problem tomorrow.

**Comment by MaxMorais** [comment]
Hi @paulhodel 

I do have solved the issue at my end, what do I have discovered, is that the issue is caused by the `MIN` function

**Comment by pphod** [comment]
Glad to hear that!

**Comment by harshdesai991** [comment]
@MaxMorais I am facing similar issue with MAX function. Could you please share how were you able to resolve this ?

**Comment by MaxMorais** [comment]
@harshdesai991, I had to monitor the entries change, and do the calculations by myself, without use formulas!

Even in Excel the calculation is fine, sounds that jExcel, didnt have an method to resolve all calculations!
Another thing is!

The order of the calculations in the rows are determinant!
Which means, you should put the dependencies of calculation first on rows!
